.circleci/config.yml

```diff
@@ -553,7 +553,7 @@ jobs:
           name: Add python to bash path
           command: echo "export PATH=\"$PATH:/c/python38/\"" >> $BASH_ENV
       - run-tests:
-          test_targets: "wasm2.test_ccall other.test_closure_externs other.test_binaryen_debug"
+          test_targets: "other.test_bad_triple wasm2.test_sse1 wasm2.test_ccall other.test_closure_externs other.test_binaryen_debug"
```

Comment:
It looks like `wasm2.test_sse1` is not actually tested, because the test is skipped due to lack of `d8` (which we install for linux, but not windows).

Trying to enable another test that uses native clang on windows, it fails on
```
LINK : fatal error LNK1104: cannot open file 'kernel32.lib'
clang: error: linker command failed with exit code 1104 (use -v to see invocation)
```

Reply:
Can we just skip all the native clang tests on windows?

Reply:
Sounds like you're suggesting that `requires_native_clang` that @dschuff recently added should always skip on windows for now? Might make sense, yeah.

Also, I think maybe we should remove `test_sse1` which was added here, since it isn't actually run, and so may be confusing.

Reply:
Agreed that we shouldn't explicitly add `test_sse1` here because as @kripken said it doesn't run on Windows because of d8. It's weird that you got that failure though because  `test_bad_triple` seems to be working fine on CI?

Reply:
The difference is `test_bad_triple` only builds an object file, it doesn't try to link using native clang.